src/util/exception_utils.cpp

```diff
@@ -0,0 +1,20 @@
+/*******************************************************************\
+
+Module: Exception helper utilities
+
+Author: Fotis Koutoulakis, fotis.koutoulakis@diffblue.com
+
+\*******************************************************************/
+
+#include "exception_utils.h"
+
+std::string invalid_user_input_exceptiont::what() const noexcept
```

Comment:
Ideally, there should also be a version hat can serialise into json and XML in a more structure way (in a future PR).